core/config.go

```diff
@@ -53,6 +53,25 @@ func (b BodyConfig) CreatePrompt(stdinReader io.ReadCloser) Prompt {
 	return p
 }
 
+type NewLineConfig struct {
+	BeforeVersion    int `yaml:"beforeVersion,omitempty"`
+	AfterVersion     int `yaml:"afterVersion,omitempty"`
+	BeforeComponent  int `yaml:"beforeComponent,omitempty"`
+	AfterComponent   int `yaml:"afterComponent,omitempty"`
+	BeforeHeader     int `yaml:"beforeHeader,omitempty"`
+	AfterHeader      int `yaml:"afterHeader,omitempty"`
+	BeforeFooter     int `yaml:"beforeFooter,omitempty"`
+	AfterFooter      int `yaml:"afterFooter,omitempty"`
```

Comment:
Lets be a little more explicit and add template to the name.
```suggestion
	BeforeHeaderTemplate     int `yaml:"beforeHeader,omitempty"`
	AfterHeaderTemplate      int `yaml:"afterHeader,omitempty"`
	BeforeFooterTemplate     int `yaml:"beforeFooter,omitempty"`
	AfterFooterTemplate      int `yaml:"afterFooter,omitempty"`
```